packages/server/app.js

```diff
@@ -71,7 +68,7 @@ exports.init = async () => {
   })
   graphqlServer = new ApolloServer({
     ...graph(),
-    context: contextApiTokenHelper,
+    context: buildContext,
```

Comment:
`contextApiTokenHelper` is very unintuitive as a name for a context builder function